server/src/db/scripts/23-bigints-for-all-primary-keys.sql

```diff
@@ -0,0 +1,77 @@
+-- users
+-- rename user_id to user_id_old, then add user_id bigint generated always as
+-- identity primary key using user_id_old::bigint
```

Comment:
Get rid of this comment, it's incorrect.

Also, all the messing with _old renaming in here was useless, since we don't actually use the old values in the new column.